File: compiled_lang/go/src/k10.parameters.go
package main

import "fmt"

func addNums(numbers ...int) {
	sum := 0
	for _, num := range numbers {
		sum += num
	}
	fmt.Printf("The summation is: %d.\n", sum)
}

func main() {
	fmt.Println("Sending: 5, 2, 4, 3, 6")
	addNums(5, 2, 4, 3, 6)
}

File: compiled_lang/rust/src/k10.parameters.rs
fn add_nums(numbers: &[i32]) {
    let mut sum = 0;
    for num in numbers {
        sum += num;
    }
    println!("The summation is: {sum}.");
}

fn main() {
    println!("Sending: 5, 2, 4, 3, 6");
    add_nums(&[5, 2, 4, 3, 6]);
}
